app/src/main/res/layout/widget_button.xml

```diff
@@ -0,0 +1,49 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:minHeight="40dp"
+    android:minWidth="40dp"
+    android:orientation="vertical"
+    android:padding="@dimen/widget_margin">
+
+    <FrameLayout
+        android:id="@+id/widgetImageButtonFrameLayout"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1">
+
+        <ImageView
+            android:id="@+id/widgetImageButtonBackground"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_margin="0dp"
+            android:contentDescription="@string/widget_image_description"
+            android:src="@drawable/ic_circle_white_24dp"
+            android:scaleType="fitCenter"
+            />
+
+        <ImageButton
+            android:id="@+id/widgetImageButton"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_margin="0dp"
+            android:contentDescription="@string/widget_image_description"
+            android:background="@null"
+            android:src="@drawable/ic_lightbulb_outline_black_24dp"
```

Comment:
Not sure a lightbulb is generic enough for a widget that calls any service.  Maybe something that indicates a more general action?

Reply:
Hrm, true.  I am thinking of my personal use case, which is why I chose a light bulb.  I would actually prefer to have a list of icons to choose from, eventually.  For now, I will look at changing it to a more generic "execute action" icon, and perhaps add a few options for specific icons (light bulb, switch, automation icon, etc)

Reply:
I added several options for this in commit 93d1570.  There's now a drop down menu for selecting an icon, and the default is the lightning bolt/flash/'action'/switch icon used by the Lovelace web interface.

Screenshot:
![Screenshot_20200117-121930_Home Assistant](https://user-images.githubusercontent.com/8548596/72632525-756e7300-3924-11ea-9f12-4a3fa80c4eac.jpg)